plugin/src/test/java/org/opensearch/ml/rest/RestCohereInferenceIT.java

```diff
@@ -74,7 +74,12 @@ public void test_cohereInference_withDifferent_postProcessFunction() {
             TextDocsInputDataSet inputDataSet = TextDocsInputDataSet.builder().docs(List.of("hello", "world")).build();
             MLInput mlInput = MLInput.builder().inputDataset(inputDataSet).algorithm(FunctionName.TEXT_EMBEDDING).build();
             Map inferenceResult = predictTextEmbeddingModel(modelId, mlInput);
-            log.info("Current post process function is: {}, inferenceResult: {}", postProcessFunction, org.opensearch.ml.common.utils.StringUtils.gson.toJson(inferenceResult));
+            log
+                .info(
```

Comment:
if this log is only for debug, we should't expose this log as info

Reply:
It doesn't seem debug log is been logging when running ITs, and this is a IT log which won't have impact on online performance. Also this is been removed as this log looks good when the test case if failing, added another more specific logs to locate the issue.